signingscript/docker.d/passwords.yml

```diff
@@ -446,6 +452,18 @@ in:
              {"$eval": "AUTOGRAPH_GPG_PASSWORD"},
              ["autograph_gpg"]
             ]
+          - ["https://autograph-external.prod.autograph.services.mozaws.net",
+             {"$eval": "AUTOGRAPH_XPI_USERNAME"},
+             {"$eval": "AUTOGRAPH_XPI_PASSWORD"},
+             ["webextension"],
+             "webextensions_rsa"  # unprivileged xpi signing
+            ]
+          - ["https://autograph-external.prod.autograph.services.mozaws.net",
+             {"$eval": "AUTOGRAPH_XPI_USERNAME"},
+             {"$eval": "AUTOGRAPH_XPI_PASSWORD"},
+             ["privileged_webextension"],
+             "extension_rsa"  # privileged xpi signing
+            ]
```

Comment:
Let's be careful not to abuse this, the mozilla-extensions pipeline should be used instead generally.